package.json

```diff
@@ -24,22 +24,25 @@
     "node": ">=18"
   },
   "browserslist": [
-    "and_chr 103",
-    "and_ff 101",
-    "and_qq 10.4",
-    "and_uc 12.12",
-    "android 103",
-    "chrome 102",
-    "edge 102",
-    "firefox 91",
-    "ios_saf 12.2-12.5",
+    "and_chr 109",
+    "and_ff 109",
+    "and_qq 13.1",
+    "and_uc 13.4",
+    "android 109",
+    "chrome 108",
+    "edge 108",
+    "firefox 102",
+    "ios_saf 16.0",
+    "ios_saf 15.6",
+    "ios_saf 14.5-14.8",
     "kaios 2.5",
     "op_mini all",
-    "op_mob 64",
-    "opera 88",
-    "safari 15.5",
-    "samsung 17.0",
-    "samsung 16.0",
+    "op_mob 73",
+    "opera 94",
+    "safari 16.2",
+    "safari 15.6",
+    "samsung 19.0",
+    "samsung 18.0",
     "node 18.0"
   ],
```

Comment:
This could be simplified to `"browserslist": "defaults"` (or something including Node)

Reply:
The thing is that having `"browserslist": "defaults"` isn't really explicit from our end, it relies on the `browserslist` and `caniuse` versions which are an implicit dependency for us.
I prefer having a closed list of browsers for two reasons:
1. We define them explicitly. When using `defaults` we have an implicit broweserslist which will be hard to debug/understand later.
2. We need to use `overrides` which is an `npm` feature. That's fine as an idea but I think that this feature is mostly used to workaround issues and not as a preferred solution.

@timdeschryver @eps1lon do you have any opinion here?

Reply:
Changing my mind now that I realize we've had the hard coded Browserslist longer than the overrides. Not my favorite solution but I think it's a reasonable workaround for now considering we don't commit lockfiles (I'm rethinking if we should still do this but will probably open an issue/pull) and we need a major release anyway.

Reply:
I agree  👍